site-src/contributing/index.md

```diff
@@ -22,7 +22,7 @@ Gateway API and not the API specification itself, please check our
 you can get help with your specific implementation.
 
 [gh-issues]: https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
-[implementations]:https://gateway-api.sigs.k8s.io/implementations/
```

Comment:
Not sure how this happened, but this change doesn't work properly.

Reply:
Good catch, thanks! I did a global find and replace for absolute links to transform them into relative ones but clearly messed up some places in the process, should be good to go now.